COPYRIGHT.txt

```diff
@@ -0,0 +1,13 @@
+Copyright 2022 The Sigstore Authors.
```

Comment:
Copyright tweak maybe? 🙂

Reply:
I didn't really change anything here, so I think this probably should retain the original copyright?

Reply:
Ah yeah, good point. I'll resolve the other comments that say the same thing.